### ISSUE TITLE
Batch size >=2 is needed 

### ISSUE THREAD

**Issue by QiaoranC** [GeeshangXu/mask-rcnn-pytorch#3]
Hi, i know Batch size is not supported 2 yet.

I just want to say the version of mask rcnn is most understandable for me.
For mask rcnn, i used that famous tf+keras version(good inference/example/document), i tried that on my own dataset, and make some progress, but the nature of keras is huge memory consumer and modified of some part (like callback) is not clear for me. So it not efficient for me. 
So now i move on pytorch mask rcnn called detection, that parallel computing looks smart but not easily to use.

So i just want to say, your vision of mask rcnn is very clear, the mask rcnn is like pytorch 'standard' model rather than 'a series of models'.
I definitely want to give a try if you can support multiple batch_size/gpus. 

So i dont really have a issue, i just want to say please continue work on this. And i am looking forward it.

**Comment by GeeshangXu** [comment]
@QiaoranC 这些正是我的目标，PyTorch版本更新很快，1.0版本与Caffe2合并，打通research与deployment，很是让人激动，我会逐步跟进的。
